src/Color/Color.scss

```diff
@@ -1,4 +1,35 @@
-.color {
-  height: 100px;
-  width: 100px;
+#color {
+  height: 200px;
```

Comment:
maybe some how these could be percentages to help with my comment above about the scaling?